tools/bumper/gitFakes.go

```diff
@@ -0,0 +1,305 @@
+package main
```

Comment:
the name of this module should be git_fakes to match naming in the rest of the project

Reply:
so git_fakes will be in a separate module/folder, right?

Reply:
Oh sorry, I mean "this file".

Reply:
Done